systemd/openqa-worker.slice

```diff
@@ -0,0 +1,5 @@
+[Unit]
+Description=Slice for openqa-worker units
+
+[Slice]
+MemoryMax=90%
```

Comment:
Please put your commit message description in here as the explanation would serve well for readers of the service file

Reply:
I added the comment, is this what you meant?